### PR TITLE
Adds authors, improves install instructions

### LINKED ISSUE #34: Add authors information?
Aside from the welcome lecture there's no list of all the authors for the workshop.

I'd be good to add this somewhere, maybe an `AUTHOR.md` or directly in the `README`?

### PR DESCRIPTION
**Pull request by IAlibay**
Towards v1.1.0
Fixes #34

Work done in this PR:
* Adds author information
* Separates out the install instructions and fixes the activation of the jupyter nbextensions (old instructions would have had them be activate in `base` rather than a specific conda environment).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,30 +117,23 @@ The workshop is structured in the following manner:
...
 
 
-
 ## Setting up your python environment
 
-In order to access this material, several python packages must be installed. A full
-list can be seen inside `environment.yml`.
-
-To set up the environment (using conda), the following can be done:
+Instructions for seting up your environment to run this workshop locally
```
Comment (lilyminium) [inline README.md]:
```suggestion
Instructions for setting up your environment to run this workshop locally

```

File: README.md [hunk 2]
```diff
@@ -117,30 +117,23 @@ The workshop is structured in the following manner:
...
+are providing in [`INSTALL.md`](INSTALL.md).
```
Comment (lilyminium) [inline README.md]:
```suggestion
are provided in [`INSTALL.md`](INSTALL.md).

```

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Installing the workshop environment
+
+## Overview
+
+This workshop is delivered mainly through a series of [Jupyter notebooks][1]
+that allow for interactive python programming.
```
Comment (lilyminium) [inline INSTALL.md]:
```suggestion
that allow for interactive Python programming.
```

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+To use these, several python packages must be installed. A full list of these
```
Comment (lilyminium) [inline INSTALL.md]:
```suggestion
To use these, several Python packages must be installed. A full list of these
```

File: INSTALL.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+can be found in [`environment.yml`](environment.yml). See below for
+instructions on how to set up a local copy of the python environment.
```
Comment (lilyminium) [inline INSTALL.md]:
```suggestion
instructions on how to set up a local copy of the Python environment.
```

File: INSTALL.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+**Note: at this time, the workshop requires the beta version of MDAnalysis 2.0.0.**
+
+
+## Installing the python environment
```
Comment (lilyminium) [inline INSTALL.md]:
```suggestion
## Installing the Python environment
```

File: INSTALL.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+Due to the complexity of the workshop environment, we strongly recommend the
+use of the [anaconda python distribution][2]. Our instructions, provided here,
```
Comment (lilyminium) [inline INSTALL.md]:
```suggestion
use of the [Anaconda Python distribution][2]. The instructions provided here
```

File: INSTALL.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+assume the use of [conda][3].
+
+### 1. Creating a workshop environment
+
+To create an environment named `mda-workshop2021` with all the necessary
+python dependencies:
```
Comment (lilyminium) [inline INSTALL.md]:
```suggestion
Python dependencies:
```

File: INSTALL.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+```bash
+conda env create --name mda-workshop2021 --file=environments.yml
+```
+
+See the [conda documentation][4] for more information on how to access and
+manage [conda][3] environments.
+
+### 2. Activating the jupyter extensions
```
Comment (lilyminium) [inline INSTALL.md]:
```suggestion
### 2. Activating the Jupyter extensions
```

File: INSTALL.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+The workshop leverages the extended utility of several jupyter nbextensions.
```
Comment (lilyminium) [inline INSTALL.md]:
```suggestion
The workshop leverages the extended utility of several Jupyter nbextensions.
```

File: README.md [hunk 11]
```diff
@@ -117,30 +117,23 @@ The workshop is structured in the following manner:
...
-jupyter nbextension enable splitcell/splitcell
-jupyter nbextension enable rubberband/main
-jupyter nbextension enable exercise2/main
-jupyter nbextension enable autosavetime/main
-jupyter nbextension enable collapsible_headings/main
-jupyter nbextension enable codefolding/main
-jupyter nbextension enable limit_output/main
-jupyter nbextension enable toc2/main
-```
+A full list of the required python packages can be seen inside [`environment.yml`](environment.yml).
```
Comment (lilyminium) [inline README.md]:
```suggestion
A full list of the required Python packages can be seen inside [`environment.yml`](environment.yml).

```

File: README.md [hunk 12]
```diff
@@ -117,30 +117,23 @@ The workshop is structured in the following manner:
...
 
 **Note: this workshop uses the beta release of MDAnalysis 2.0.0.**
 
 
+## Course pre-requisites
+
+The course assumes that attendees have a working proficiency in using
+[jupyter notebooks][1], python (especially the [numpy library][2]), and the
```
Comment (lilyminium) [inline README.md]:
```suggestion
[Jupyter notebooks][1], Python (especially the [NumPy library][2]), and the

```

File: README.md [hunk 13]
```diff
@@ -154,3 +147,17 @@ license. Non-code content is licensed under the Creative Commons
 Attribution-ShareAlike 4.0 International license. The MDAnalysis logo and its
 derivatives are licensed under the Creative Commons Attribution-NoDerivs 3.0
 Unported License.
+
+
+## Acknowledgements
+
+Please see [`AUTHORS.md`](AUTHORS.md) for a list contributors to the workshop
```
Comment (lilyminium) [inline README.md]:
```suggestion
Please see [`AUTHORS.md`](AUTHORS.md) for a list of contributors to the workshop

```

**Review by lilyminium — APPROVED** [review]
Sorry, I'm a pedant :p

But otherwise this looks great, thanks @IAlibay for putting in the work

**Comment by IAlibay** [comment]
> Sorry, I'm a pedant :p

Not at all, I suspected there were a ton of typos, I'm glad someone spotted them.

**Pull request merged**